packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/transforms.js

```diff
@@ -364,6 +364,58 @@ describe('EDiscovery Transform Tests', () => {
 
         return result;
       });
+
+      it('Calls the correct decrypt functions when transforming activity.spaceInfo.name', () => {
+        object.body.verb = 'update';
+        object.body.spaceInfo = {name: 'Encrypted Name'};
+        object.body.objectDisplayName = undefined;
+        const result = Transforms.decryptReportContent(ctx, object, reportId)
+          .then(() => {
+            assert.callCount(ctx.webex.internal.encryption.decryptText, 1);
+            assert.equal(activity.error, undefined);
+          });
+
+        return result;
+      });
+
+      it('Calls the correct decrypt functions when transforming activity.spaceInfo.name with previousValue', () => {
+        object.body.verb = 'update';
+        object.body.spaceInfo = {name: 'Encrypted Name', previousName: 'Previous Name', previousEncryptionKeyUrl: keyUri};
+        object.body.objectDisplayName = undefined;
+        const result = Transforms.decryptReportContent(ctx, object, reportId)
+          .then(() => {
+            assert.callCount(ctx.webex.internal.encryption.decryptText, 2);
+            assert.equal(activity.error, undefined);
+          });
+
+        return result;
+      });
+
+      it('Calls the correct decrypt functions when transforming activity.spaceInfo.description', () => {
+        object.body.verb = 'update';
+        object.body.spaceInfo = {description: 'Encrypted Description'};
```

Comment:
Looks like the test does not require `descriptionEncryptionKeyUrl` to be set. That is not a blocker, it simply shows the test can't be too realistic.

Reply:
Since this is a description _activity_, there is no descriptionEncryptionKeyUrl,  the encryptionKeyUrl will be the correct property to decrypt the description. For a _space_ it's different, there will be two urls, one for name and one for description.